pkg/minikube/download/preload.go

```diff
@@ -141,6 +142,12 @@ func Preload(k8sVersion, containerRuntime string) error {
 	out.Step(style.FileDownload, "Downloading Kubernetes {{.version}} preload ...", out.V{"version": k8sVersion})
 	url := remoteTarballURL(k8sVersion, containerRuntime)
 
+	checksum, err := getChecksum(k8sVersion, containerRuntime)
+	if err != nil {
+		return errors.Wrap(err, "getting checksum")
```

Comment:
should we return error here or should just at least try to download it regardless of being able to get checksum ?

Reply:
It will fail a couple of lines down anyway, when it tries to save the same checksum to a file.

Reply:
I don't know how unrealiable Google Cloud Storage, if it usually fails when trying to get metadata ?

Reply:
Implemented the version without a checksum up front, doing it all manually instead (tempfile + verify)